suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -1153,3 +1155,59 @@ export const isTokenMatchesSearch = (token: TokenInfo, search: string) => {
         token.policyId?.toLowerCase().includes(search)
     );
 };
+
+export const isDebugOnlyAccountType = (
+    accountType: AccountType,
+    symbol?: NetworkSymbol,
+): boolean => {
+    if (!symbol) return false;
+
+    const network = (networks as Networks)?.[symbol];
+
+    if (!network) return false;
```

Comment:
:bulb: This can be simplified without `as`, and without the `if`.
Btw I added `wallet-config/getNetwork` only recently, before it had to be like this, but now it can be nicer :blush: 
```suggestion
    const network = getNetwork(symbol);
```

Reply:
Thanks, good to know